typescript/forms.d.ts

```diff
@@ -197,6 +195,8 @@ export type FormElementWithOptionsBase = LookupFormElement &
      * element ID within the form definition
      */
     attributesMapping?: DynamicOptionsSetAttributeMap[]
+    /** Custom autocomplete attributes that will be added to the element during rendering */
+    autocompleteAttributes?: string[]
```

Comment:
Do we want it here? I can't see the autocomplete working for `SELECT`, `RADIO`, `CHECKBOXES`, or `AUTOCOMPLETE`. It might be good for `FRESHDESK` but I can't remember if that shows options.

Reply:
If we want it on autocomplete/freshdesk, might be better to include it on them explicitly

Reply:
It will work for selects and autocomplete, not sure about radio or checkboxes.

Reply:
I guess we can leave it for now and update the type if it does not work for radio/checkboxes